back-end/Recognition/src/utils.py

```diff
@@ -0,0 +1,49 @@
+from google.cloud import vision
+
+def process_image(image_path):
+    """
+    Processes an image file using the Google Vision API to perform text detection and label detection.
+
+    Args:
+        image_path (str): The file path of the image to process.
+
+    Returns:
+        tuple: A tuple containing the text annotations and label words as dictionaries.
+
+    Raises:
+        FileNotFoundError: If the image file cannot be found.
+
+    Example:
+        >>> image_path = 'C:/Users/nasser/Downloads/test.jpg'
+        >>> texts, label_words = process_image(image_path)
+    """
+
+    # Instantiate the Google Vision client
+    client = vision.ImageAnnotatorClient() # Comment it after Reema's code
```

Comment:
dont init the client here 
remove these lines and in the top of the folder write 
from Road import client 
and suppose that rema already init the client in  the __init__.py file

Reply:
Road?